app/managers/performance_metrics_manager/index.ts

```diff
@@ -104,6 +106,22 @@ class PerformanceMetricsManager {
         performance.clearMarks(metricName);
         performance.clearMeasures(measureName);
     }
+
+    public startTimeToInteraction(options?: MarkOptions) {
+        performance.mark('tti', options);
+    }
+
+    public measureTimeToInteraction() {
```

Comment:
Yay, long anticipated metric is here :)

Reply:
yes, but is nothing more than an output to the console at the moment.. I need to figure out where and how to include it.

Reply:
Should I create another metric on the server to collect this?

Reply:
@rahimrahman can it be optional? I think that'll be very nice to have but haven't checked into how to include it as is a totally different metric collected in a different way and somewhere else in the code